core/gesture.js

```diff
@@ -836,8 +836,11 @@ Blockly.Gesture.prototype.duplicateOnDrag_ = function() {
   try {
     // Note: targetBlock_ should have no children.  If it has children we would
     // need to update shadow block IDs to avoid problems in the VM.
+    // Resizes will be reenabled at the end of the drag.
+    this.startWorkspace_.setResizesEnabled(false);
```

Comment:
Is this a different fix?

Reply:
Yup, just avoiding a resize during the user action.